docs/tutorial/part-three/index.md

```diff
@@ -1,104 +1,104 @@
+[_Typography.js_](https://kyleamathews.github.io/typography.js/) adalah sebuah perpustakaan dari Javasript yang menghasilkan bentuk dasar tampilan dari _typography_ secara keseluruhan untuk situs anda. Perpustakaan ini memiliki sebuah [_plugin_ dari Gatsby yang sesuai](/packages/gatsby-plugin-typography/) yang mempermudah penggunaannya di situs Gatsby anda.
 
-### ✋ Create a new Gatsby site
+### ✋ Membuat sebuah situs baru menggunakan Gatsby
 
-As we mentioned in [part two](/tutorial/part-two/), at this point it's probably a good idea to close the terminal window(s) and project files from previous parts of the tutorial, to keep things clean on your desktop. Then open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-three` and then move to this new directory:
+Seperti yang kami sebutkan di [bagian kedua](/tutorial/part-two/), pada titik ini alangkah baiknya dengan menutup jendela terminal yang tampil kemudian memproyeksikan file dari bagian tutorial sebelumnya, untuk menjaga semuanya tetap bersih pada bagian Desktop anda. lalu buka jendela terminal baru dan jalankan perintah berikut untuk membuat situs Gatsby baru didalam direktori yang disebut `tutorial-bagian-tiga` dan kemudian pindah ke direktori baru ini:
 
 ```shell
 gatsby new tutorial-part-three https://github.com/gatsbyjs/gatsby-starter-hello-world
 cd tutorial-part-three
 ```
 
-### ✋ Install and configure `gatsby-plugin-typography`
+### ✋ Menambahkan dan konfigurasi `gatsby-plugin-typography`
 
-There are two main steps to using a plugin: Installing and configuring.
+Ada dua langkah utama untuk menggunakan _plugin_: Menambahkan dan konfigurasi.
 
-1. Install the `gatsby-plugin-typography` NPM package.
+1. Menambahkan _package_ dari _NPM_ `gatsby-plugin-typography`.
 
 ```shell
 npm install --save gatsby-plugin-typography react-typography typography typography-theme-fairy-gates
 ```
 
-> Note: Typography.js requires a few additional packages, so those are included in the instructions. Additional requirements like this will be listed in the "install" instructions of each plugin.
+> Catatan: _Typography.js_ memerlukan beberapa _packages_ tambahan, itu termasuk dalam instruksi. Persyaratan tambahan seperti ini akan tercantum dalam instruksi "_install_" dari masing-masing penggunaan _plugin_.
 
-2. Edit the file `gatsby-config.js` at the root of your project to the following:
+2. Lakukan perubahan pada file `gatsby-config.js` yang berada pada struktur bagian utama proyek anda dengan cara berikut:
 
 ```javascript:title=gatsby-config.js
 module.exports = {
   plugins: [
     {
       resolve: `gatsby-plugin-typography`,
       options: {
-        pathToConfigModule: `src/utils/typography`,
-      },
-    },
-  ],
-}
+        pathToConfigModule: `src/utils/typography`
```

Comment:
Dont ever change the code, please revert it back 😉

Reply:
hedehh linter vscode :(
otewe perbaikan mas

Reply:
Ashiap @herkahahaha , take your time